addons/snailmail_account/wizard/account_invoice_send.py

```diff
@@ -16,14 +16,17 @@ class AccountInvoiceSend(models.TransientModel):
         default=lambda self: self.env.company.invoice_is_snailmail)
     snailmail_cost = fields.Float(string='Stamp(s)', compute='_compute_snailmail_cost', readonly=True)
     invalid_addresses = fields.Integer('Invalid Addresses Count', compute='_compute_invalid_addresses')
-    invalid_invoice_ids = fields.Many2many('account.move', string='Invalid Addresses', compute='_compute_invalid_addresses')
+    invalid_invoices = fields.Integer('Invalid Invoices Count', compute='_compute_invalid_addresses')
+    invalid_partner_ids = fields.Many2many('res.partner', string='Invalid Addresses', compute='_compute_invalid_addresses')
 
     @api.depends('invoice_ids')
     def _compute_invalid_addresses(self):
         for wizard in self:
-            invalid_invoices = wizard.invoice_ids.filtered(lambda i: not i.partner_id or not self.env['snailmail.letter']._is_valid_address(i.partner_id))
-            wizard.invalid_invoice_ids = invalid_invoices
-            wizard.invalid_addresses = len(invalid_invoices)
+            invalid_invoices = wizard.invoice_ids.filtered(lambda i: not self.env['snailmail.letter']._is_valid_address(i.partner_id))
+            wizard.invalid_invoices = len(invalid_invoices)
+            invalid_partner_ids = list(set([i.partner_id.id for i in invalid_invoices]))
```

Comment:
you can use set comprehension to avoid creating a list that you convert into a set that you convert into a list.
```
invalid_partner_ids = list({i.partner_id.id for i in invalid_invoices})
```